docker-compose.override.yml

```diff
@@ -1,122 +1,122 @@
-  backend:
-    # name the image so that it cannot be found in a DockerHub repository, otherwise it will not be built locally from the 'dockerfile' but pulled from there
-    image: gradido/backend:local-development
-    build:
-      target: development
-    networks:
-      - external-net
-      - internal-net
-    environment:
-      - NODE_ENV="development"
-    volumes:
-      # This makes sure the docker container has its own node modules.
-      # Therefore it is possible to have a different node version on the host machine
-      - backend_node_modules:/app/node_modules
-      - backend_database_node_modules:/database/node_modules
-      - backend_database_build:/database/build
-      # bind the local folder to the docker to allow live reload
-      - ./backend:/app
-      - ./database:/database
-
-  ########################################################
-  # DATABASE ##############################################
-  ########################################################
-  database:
-    # we always run on production here since else the service lingers
-    #  feel free to change this behaviour if it seems useful
-    #  Due to problems with the volume caching the built files
-    #  we changed this to test build. This keeps the service running.
-    # name the image so that it cannot be found in a DockerHub repository, otherwise it will not be built locally from the 'dockerfile' but pulled from there
-    image: gradido/database:local-test_up
-    build:
-      target: test_up
-    environment:
-      - NODE_ENV="development"
-    volumes:
-      # This makes sure the docker container has its own node modules.
-      # Therefore it is possible to have a different node version on the host machine
-      - database_node_modules:/app/node_modules
-      - database_build:/app/build
-      # bind the local folder to the docker to allow live reload
-      - ./database:/app
-
-  #########################################################
-  ## MARIADB ##############################################
-  #########################################################
-  mariadb:
-    networks: 
-      - internal-net
-      - external-net
-      
-  #########################################################
-  ## NGINX ################################################
-  #########################################################
-  # nginx:
-  
-  #########################################################
-  ## PHPMYADMIN ###########################################
-  #########################################################
-  phpmyadmin:
-    image: phpmyadmin
-    environment:
-     - PMA_ARBITRARY=1
-    #restart: always
-    ports:
-     - 8074:80
-    networks: 
-     - internal-net
-     - external-net
-    volumes:
-     - /sessions
-
-volumes:
-  frontend_node_modules:
-  admin_node_modules:
-  backend_node_modules:
-  backend_database_node_modules:
-  backend_database_build:
-  database_node_modules:
-  database_build:
+version: "3.4"
```

Comment:
Check your WS. We need UNIX (`x0A`) instead of windows (`x0Dx0A`)

Reply:
yes sorry.
how can we prepaire it for the root directory? 
Because we've done it in the different submodules by adding in the packages.json scripts the entry cross-env.

Reply:
Try `dos2unix`. This way we cannot merge the files.

Reply:
yes sorry, is converted now...

on win git always makes a checkout with 0x0D0x0A, my IDE works with both WIN or UNIX, but i must manually change it to UNIX format.
i try to find a solution to get the fils directly in UNIX format...